crates/toml/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 //! ```toml
 //! [package]
 //! name = "toml"
-//! version = "0.4.2"
+//! version = "0.7.1"
 //! authors = ["Alex Crichton <alex@alexcrichton.com>"]
 //!
 //! [dependencies]
```

Comment:
This is meant to be a real world example of TOML.  I do not see the value in creating churn in updating the version on every release though.

Reply:
Okay :smiley: